examples/doubleclick-usdrud.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Doubleclick Example with UseSameDomainRenderingUntilDeprecated</title>
+  <link rel="canonical" href="http://nonblocking.io/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-custom>
```

Comment:
I don't think any of the CSS is necessary right?

Reply:
removed